### ISSUE TITLE
POD NOTES: Publish to Google Play Store

### ISSUE THREAD

**Issue by gjwgit** [anusii/podnotes#110]

**Comment by jesscmoore** [comment]
I'll start testing on android. Id like to resolve the title not always loading on external notes issue, then keen to see it published soon.

@gjwgit @anushkavidanage   do  you have other steps you'd like done before publication?

It would be good to have tests in place. Is there a way we can implement tests that test notes are found for a test account?

**Comment by gjwgit** [comment]
Yes, I think this is ready for publication. I use the app on my android already - it is quite handy.

Test suites would be good. @atangster developed some integration tests for healthpod, including logging in to accounts if a user login is cached (as I recall). We could do something similar here. Also unit tests in moviestar. Rattle has integration tests that take some 90 minutes to run - so some further examples there.

**Comment by jesscmoore** [comment]
> Yes, I think this is ready for publication. I use the app on my android already - it is quite handy.

Great to hear.

> Test suites would be good. [@atangster](https://github.com/atangster) developed some integration tests for healthpod, including logging in to accounts if a user login is cached (as I recall). 

Could I get access to see tests in healthpod or moviestar (if the latter uses cached login), then I'll add similar for notepod?

**Comment by gjwgit** [comment]
Move back to ready for now until we have ironed out a batch of current bugs.

**Comment by gjwgit** [comment]
@jesscmoore - a couple of issues I came across while registering with Google Play Store: #257 and #258

**Comment by gjwgit** [comment]
App currently broken not creating new notes: #263 - FIXED.

**Comment by gjwgit** [comment]
Fixed 2x login issue.

New version 0.2.44 pushed to Google Play Store for testing: https://play.google.com/store/apps/details?id=com.togaware.notepod
